app/src/main/java/com/picone/go4lunch/presentation/utils/BottomNavigationUtil.java

```diff
@@ -0,0 +1,31 @@
+package com.picone.go4lunch.presentation.utils;
```

Comment:
Are you passing a fragment as a parameter. Remember it is not a good practice. You should not embed Android Framework components as a class dependancy. Plus, you have to keep the UI as dumb as possible. 
There is a way to set up Navigation and BottomNavigation. I let you search around.

Reply:
Yes i know that it's not a good pratice to pass fragment as parameter, I' ve find a way to fix it.

Reply:
[https://medium.com/@freedom.chuks7/how-to-use-jet-pack-components-bottomnavigationview-with-navigation-ui-19fb120e3fb9](url)

take it look at the article